site/content/docs/5.3/components/orange-navbar.md

```diff
@@ -192,7 +192,7 @@ An additional navbar (with text or icon items) can be added on the right of the
               </svg>
               <span class="visually-hidden">Basket</span>
               <span class="position-relative align-self-start">
-                <span class="badge bg-info rounded-pill position-absolute top-0 fs-6 text-white translate-middle">
+                <span class="badge text-bg-info rounded-pill position-absolute top-0 fs-6 text-white translate-middle">
```

Comment:
There's no impact here but we will need to define in the dark mode implementation whether `.text-bg-info` should render the text color in white automatically or black. For now, `.text-white` is still needed.